clusterloader2/api/types.go

```diff
@@ -106,6 +109,19 @@ type ListUnknownObjectOptions struct {
 	LabelSelector *metav1.LabelSelector `json: labelSelector`
 }
 
+// NamespaceConfig defines namespace parameters.
+type NamespaceConfig struct {
+	// Number is a number of automanaged namespaces.
+	Number int32 `json: number,omitempty`
+	// NamePrefix is the name prefix of automanaged namespaces.
+	// It's optional, if set CL will use it, otherwise generate one with random string.
+	Prefix string `json: prefix,omitempty`
+	// DeleteStaleNamespaces specifies whether or not delete stale namepaces
+	DeleteStaleNamespaces bool `json: deleteStaleNamespaces,omitempty`
```

Comment:
As mentioned above - these bool fields should be pointers.

Reply:
Changed